mix.lock

```diff
@@ -1,14 +1,14 @@
 %{
-  "bunt": {:hex, :bunt, "0.2.0", "951c6e801e8b1d2cbe58ebbd3e616a869061ddadcc4863d0a2182541acae9a38", [:mix], [], "hexpm", "7af5c7e09fe1d40f76c8e4f9dd2be7cebd83909f31fee7cd0e9eadc567da8353"},
```

Comment:
After running `mix hex.outdated`, I noticed that everything was outdated, so, I decided to bump all the dependencies, most of them for development purposes (`ex_doc` and `credo`)